src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -658,46 +661,69 @@ def copy_files_to_arrange(sourcepath, destination):
+    logger.info('destinations: %s', destinations)
+
+    # The DEFAULT_BACKLOG_PATH constant is missing a leading slash for
+    # historical reasons; TODO change this at some point.
+    # External paths passed into these views are in the format
+    # /originals/, whereas copy_from_arrange_to_completed constructs
+    # paths without a leading slash as an implementation detail
+    # (to communicate with the Storage Service).
+    # Possibly the constant used to refer to externally-constructed
+    # paths and the one used solely internally should be two different
+    # constants.
+    if sources[0].startswith('/' + DEFAULT_BACKLOG_PATH):
+        action = 'copy'
+        backlog_uuid = storage_service.get_location(purpose='BL')[0]['uuid']
+    elif sources[0].startswith(DEFAULT_ARRANGE_PATH):
+        action = 'move'
+    else:
+        logger.error('Filepath %s is not in base backlog path nor arrange path')
+        return helpers.json_response(
+            {'error': True, 'message': '{} is not in base backlog path nor arrange path'.format(sources[0])}
+        )
 
     try:
-        # The DEFAULT_BACKLOG_PATH constant is missing a leading slash for
-        # historical reasons; TODO change this at some point.
-        # External paths passed into these views are in the format
-        # /originals/, whereas copy_from_arrange_to_completed constructs
-        # paths without a leading slash as an implementation detail
-        # (to communicate with the Storage Service).
-        # Possibly the constant used to refer to externally-constructed
-        # paths and the one used solely internally should be two different
-        # constants.
-        if sourcepath.startswith('/' + DEFAULT_BACKLOG_PATH):
-            copy_files_to_arrange(sourcepath, destination)
-            response = {'message': 'Files added to the SIP.'}
-            status_code = 201
-        elif sourcepath.startswith(DEFAULT_ARRANGE_PATH):
-            move_files_within_arrange(sourcepath, destination)
-            response = {'message': 'SIP files successfully moved.'}
-            status_code = 200
-        else:
-            raise ValueError('filepath is in neither the base backlog path nor the arrange path ({} provided)'.format(sourcepath))
+        for source, dest in zip(sources, destinations):
+            if action == 'copy':
+                copy_files_to_arrange(source, dest,
+                    fetch_children=fetch_children, backlog_uuid=backlog_uuid)
+                response = {'message': 'Files added to the SIP.'}
+                status_code = 201
+            elif action == 'move':
+                move_files_within_arrange(source, dest)
+                response = {'message': 'SIP files successfully moved.'}
+                status_code = 200
     except ValueError as e:
-        logger.exception('copy_to_arrange failed copying %s to %s', (sourcepath, destination))
+        logger.exception('Failed copying %s to %s', (source, dest))
```

Comment:
I think the syntax is off here, no? Shouldn't it be one of the following:

    `logger.exception('Failed copying %s to %s', source, dest)`
    `logger.exception('Failed copying %s to %s' % (source, dest))`